docs/content/integrations/dbt.mdx

```diff
@@ -58,6 +58,26 @@ def my_dbt_pipeline():
 
 In the code snippet above, the config `"models"` takes a list of strings. The string `"tag:staging"` uses [dbt's node selection syntax](https://docs.getdbt.com/reference/node-selection/syntax) to filter models with the tag `"staging"`. For more details, [visit the official dbt documentation on the node selection syntax](https://docs.getdbt.com/reference/node-selection/syntax).
 
+**Example:** The solid `dbt_cli_run` is configured to run specific models similar to above except after another solid has executed. When use the `start_after` argument, which has the `InputDefinitions` of `dagster_type=Nothing`, you can assign the output from another solid so that you can still model the dependencies with another solid. See the [Order Based Dependencies](/concepts/solids-pipelines/pipelines#order-based-dependencies-nothing-dependencies) for more info.
+```python file=/integrations/dbt.py startafter=start_marker_dbt_cli_run_after_another_solid endbefore=end_marker_dbt_cli_run_after_another_solid dedent=4
```

Comment:
I don't see the corresponding change for this snippet in dagster/examples/docs_snippets/docs_snippets/integrations/dbt.py. These snippets are autogenerated (using the yarn snapshot command), and you can look in that file to see some examples (or maybe you just forgot to add those changes to this diff).

Reply:
Nope. Just had no idea what I was supposed to do. Will get this fixed.